### ISSUE TITLE
AWS_DEFAULT_REGION ignored since v0.12.7

### ISSUE THREAD

**Issue by jansepke** [localstack/localstack#3662]
# Type of request: This is a ...

[x ] bug report
[ ] feature request

# Detailed description

We are using AWS_DEFAULT_REGION=eu-central-1 env variable to set a different region for local stack. this stopped working after upgrading from 0.12.6 to 0.12.7. We now need to also set DEFAULT_REGION=eu-central-1.

As far as I can see this commit https://github.com/localstack/localstack/commit/c092f9fea237f6947d597d2a6b032b535763e237 introduced this change:
https://github.com/localstack/localstack/blob/b07df43b1ea2291595a2b112e533a42d114f1a3c/localstack/dashboard/infra.py#L41
But as DEFAULT_REGION is always initialized with AWS_REGION_US_EAST_1:
https://github.com/localstack/localstack/blob/b07df43b1ea2291595a2b112e533a42d114f1a3c/localstack/config.py#L43
it is now not possible anymore to override the region by setting only AWS_DEFAULT_REGION.

Or should we always use DEFAULT_REGION instead of AWS_DEFAULT_REGION?

## Expected behavior

AWS_DEFAULT_REGION=eu-central-1 changes the region to eu-central-1

## Actual behavior

region is us-east-1

**Comment by mgagliardo** [comment]
Hello @jansepke, thanks for reporting this.
* TL; DR: Please use`DEFAULT_REGION` as env var, it should work properly.

* Long run:  Yes, this changed on the [latest release](https://github.com/localstack/localstack/releases), you can see [here](https://github.com/localstack/localstack/blob/b07df43b1ea2291595a2b112e533a42d114f1a3c/localstack/config.py#L42) is where it grabs the env var.

**Comment by jansepke** [comment]
hm, this is a bit missleading as AWS docs prefer AWS_DEFAULT_REGION: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html

but if DEFAULT_REGION is the way to got for localstack at least the changelog could be improved
"fix setting of default region from environment variables" (under Bug Fixes)
should be changed to a breaking change...

**Comment by mgagliardo** [comment]
What you are referring to is the AWS CLI docs, which is separated from bringing up a cluster with localstack.
I agree it could be modified on the `CHANGELOG.md` though, opinions @whummer ?

**Comment by jansepke** [comment]
I just noticed that `DEFAULT_REGION` is clearly in the documentation and has been there for years. So I would say we should read it next time completely ;) We thought that localstack would use the same ENV vars like AWS CLI but you are right that is something else.
